cargo-cyclonedx/src/generator.rs

```diff
@@ -87,21 +88,21 @@ impl SbomGenerator {
             log::trace!("Config from config override: {:?}", config_override);
             log::debug!("Config from merged config: {:?}", config);
 
-            let dependencies =
+            let (dependencies, _resolve) =
```

Comment:
Is this unused in preparation for the dependency tree?

Reply:
Exactly.